### DISCUSSION TITLE
YouTubePlus [RELEASE]

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/YouTubeRebornPlus#23] [General]
Introducing a different version of CercubePlusExtra. it has all of the features from CercubePlusExtra except any option related to Cercube and no Cercube deb file.
this tweak is called YouTubePlus! DOES NOT include the tweaks uYou or Cercube. just a plain YouTube ipa with the Features you known and loved from CercubePlusExtra. just without the Cercube related options.

Reason: the whole reason I made this is for people that don't want to be forced to use the Cercube Tweak or the uYou Tweak that are included in the ipa and this makes YT+ more stable without any of those tweaks/debs included.

DOWNLOAD: https://github.com/arichorn/YouTubePlus/releases/latest

**Comment by nexus8x** [comment]
Does it have the option to set default video quality?  I only need this when using Cercube

**Reply by arichornlover** [reply]
> Does it have the option to set default video quality? I only need this when using Cercube

well, I completely forgot about that being important. So my recommendation is that you don’t have to switch to that.

**Reply by nexus8x** [reply]
> > Does it have the option to set default video quality? I only need this when using Cercube
> 
> well, I completely forgot about that being important. So my recommendation is that you don’t have to switch to that.

I'm looking forward to this feature..for me just simply block ads and set default video quality..a gentle YouTube like this YT+..God bless you
